activitystreams-core/index.html

```diff
@@ -1162,6 +1162,221 @@
        favor the use of properties defined by the Activity Vocabulary.
      </p>
 
+     <p>
+        Similarly, it is possible use multiple vocabularies to cover particular
+        characteristics of the activities like data provenance and annotations,
+        which can compliment the Activity Vocabulary. For example: Eric writes a
+        short note to be shared with his followers. After posting the note, he
+        notices a spelling error. He edits the note and re-posts it. Later, Eric
+        decides that the information in the note is incorrect. He deletes the
+        note.
+    </p>
+
+     <figure><figcaption>A series of activities; creating, editing, and deleting a note.</figcaption>
+<div class="nanotabs">
+  <ul>
+    <li><a href="#ex7-jsonld" class="selected">JSON-LD</a></li>
+    <li><a href="#ex7-rdfa" class="selected">RDFa</a></li>
+    <li><a href="#ex7-turtle" class="selected">Turtle</a></li>
+  </ul>
+  <div id="ex7-jsonld" style="display: block;">
+  <pre class="example highlight json">{
+  "@context": [
+    "http://www.w3.org/ns/activitystreams#",
+    {
+      "oa": "http://www.w3.org/ns/oa#",
+      "prov": "http://www.w3.org/ns/prov#"
+    }
+  ],
+  "@type": "Collection",
+  "items": [
+    {
+      "@id": "http://example.org/activity/20150101000000",
+      "@type": [ "Create", "prov:Activity" ],
+      "actor": {
+        "@id": "http://example.org/#eric",
+        "displayName": "Eric"
+      },
+      "displayName": "Eric wrote a note.",
+      "object": {
+        "@id": "http://example.org/entry/20150101000000",
+        "@type": [ "Note", "prov:Entity" ],
+        "attributedTo": "http://example.org/#eric",
+        "content": "Remember... all I'm offering is the trooth. Nothing more."
+      },
+      "published": {
```

Comment:
@akuckartz... if you get around to submitting a fixed PR... "published" must not use the JSON-LD value object form. It needs to be a string value